galley/pkg/config/source/kube/apiserver/source.go

```diff
@@ -184,19 +183,29 @@ func (s *Source) startWatchers() {
 	for i, r := range resources {
 		a := s.provider.GetAdapter(r)
 
+		found := s.foundResources[asKey(r.Group, r.Kind)]
+
 		scope.Source.Infof("[%d]", i)
 		scope.Source.Infof("  Source:       %s", r.CanonicalResourceName())
 		scope.Source.Infof("  Name:  		 %s", r.Collection)
 		scope.Source.Infof("  Built-in:     %v", a.IsBuiltIn())
 		if !a.IsBuiltIn() {
-			scope.Source.Infof("  Found:  %v", !r.Disabled)
+			scope.Source.Infof("  Found:  %v", found)
 		}
 
-		if a.IsBuiltIn() || !r.Disabled {
+		// Create a watcher for resources that are builtin or were found
+		if a.IsBuiltIn() || found {
```

Comment:
Also this doesn't seem to be taking disabled into account?

Reply:
Refactored these to use an if/else structure.